vit-servicing-station-tests/src/common/load/voting_power.rs

```diff
@@ -0,0 +1,54 @@
+use crate::common::clients::RestClient;
+use crate::common::snapshot::Snapshot;
+use jortestkit::load::{Request, RequestFailure, RequestGenerator};
+use rand::Rng;
+use rand::RngCore;
+use std::time::Duration;
+
+const DEFAULT_MAX_SPLITS: usize = 7; // equals to 128 splits, will likely not reach that value but it's there just to prevent a stack overflow
+
+#[derive(Clone, Debug)]
+pub struct VotingPowerRequestGenerator {
+    rest_client: RestClient,
+    snapshot: Snapshot,
+    max_splits: usize, // avoid infinite splitting
+}
+
+impl VotingPowerRequestGenerator {
+    pub fn new(snapshot: Snapshot, mut rest_client: RestClient) -> Self {
+        rest_client.disable_log();
+
+        Self {
+            snapshot,
+            rest_client,
+            max_splits: DEFAULT_MAX_SPLITS,
+        }
+    }
+}
+
+impl RequestGenerator for VotingPowerRequestGenerator {
+    fn next(&mut self) -> std::result::Result<Request, RequestFailure> {
+        let mut rng = rand::rngs::OsRng;
+        let content = &self.snapshot.content;
+        self.rest_client
+            .voting_power(
```

Comment:
what's this request for?

Reply:
for voting_power and group.. maybe i should rename it?